Gemfile

```
@@ -174,9 +174,12 @@ end
 group :production do
   # sidekiq enterprise requires a license key to download but is only required in production.
   # for local dev environments, regular sidekiq works fine
-  unless (Bundler::Settings.new['enterprise.contribsys.com'].nil? ||
-          Bundler::Settings.new['enterprise.contribsys.com']&.empty?) &&
-         ENV.fetch('BUNDLE_ENTERPRISE__CONTRIBSYS__COM', '').empty?
+  if (Bundler::Settings.new(Bundler.app_config_path)['enterprise.contribsys.com'].nil? ||
+      Bundler::Settings.new(Bundler.app_config_path)['enterprise.contribsys.com']&.empty?) &&
+     ENV.fetch('BUNDLE_ENTERPRISE__CONTRIBSYS__COM', '').empty?
+    puts 'No credentials found to install Sidekiq Enterprise. This is fine for local development but you may not check in this Gemfile.lock with any Sidekiq gems removed. The README file in this directory contains more information.'
```

We can add some color.  This will output the text in yellow:
```suggestion
    Bundler.ui.warn 'No credentials found to install Sidekiq Enterprise. This is fine for local development but you may not check in this Gemfile.lock with any Sidekiq gems removed. The README file in this directory contains more information.'
```